### DISCUSSION TITLE
Cross-platform?

### DISCUSSION THREAD

**Discussion by trypsynth** [mush42/bookworm#176] [Q&A] [answered]
I've seen some code that aims to be cross-platform in Bookworm's directory, but recently, it seems you've started using native Windows APIs for a lot of things. Is a cross-platform ebook reader no longer planned? If not, that would be quite sad, as I want to have one reader for all my platforms.

**Comment by mush42** [comment, marked as answer]
Hello @TheQuinbox 

Although Bookworm does not claim to be a cross-platform application, yet. I try to keep the core of Bookworm platform-augnostic. 

Yes. Some recent changes make use of native win32 APIs to enhance the user experience, specially with regard to the Ritch edit control. 

But those changes are specific to Windows, and as per planned changes, the appropriate class is chosen at runtime based on the current platform. This enables us to benefit from the native features provided by each platform.

Best
Musharraf
